src/Microsoft.OData.Client/SaveResult.cs

```diff
@@ -320,11 +320,6 @@ protected override DataServiceResponse HandleResponse()
                     if (response.Exception != null)
                     {
                         operationResponse.Error = response.Exception;
-
-                        if (ex == null)
-                        {
-                            ex = response.Exception;
-                        }

```

Comment:
Isn't this a breaking change? The existing clients may serialize error responses now. While I agree that throwing exception is not the right thing to do, this may lead to error responses being shown in places where it wasn't being shown.

Reply:
Spec: http://docs.oasis-open.org/odata/odata/v4.0/os/part1-protocol/odata-v4.0-os-part1-protocol.html#_Toc372793627

> The odata.continue-on-error preference on a batch request is used to request that, upon encountering a request within the batch that returns an error, the service return the error for that request and continue processing additional requests within the batch.
If not specified, upon encountering an error the service MUST return the error within the batch and stop processing additional requests within the batch.

Shouldn't the client then mimic the service's behavior such that if the request specifies the `odata.continue-on-error` preference and the service supports it then return error via COR otherwise throw the exception. Support for this can be implicitly determined by whether the entire batch request fails or not.